### ISSUE TITLE
Loading Framework

### ISSUE THREAD

**Issue by Pandemicland559** [rtakehara/5e-Framework#35]
When I open a server and join my server on a different device, it says 'loading framework' and doesn't load.

1. Start server and join
2. Wait for 'loading framework;
3. It doesn't end (never loads)

Loading Framework, never loads

![image](https://user-images.githubusercontent.com/77243261/104131384-e85ece00-536d-11eb-99f3-cfa7fdaa2aeb.png)

Desktop
 - OS: [Windows]
 - Maptool Version [1.7.0]
 - Framework Version [2.5.6]

**Comment by rtakehara** [comment]
try making the token Lib:Campaign visible to the player and see if solves it

**Comment by Sigmundx** [comment]
I am having this same issue and tried making Lib:Campaign visible but it was already set to visible to players

**Comment by SolaticSyot** [comment]
> I am having this same issue and tried making Lib:Campaign visible but it was already set to visible to players

Also having this issue and likewise Lib:Campaign was visible to players already.

**Comment by rtakehara** [comment]
Ah, my bad, it says Lib:Notebook (GM), so try making that one visible. If you want to hide it from the players, leave it on a hidden map (such as 00.GM) and be sure they don't have ownership.

**Comment by Sigmundx** [comment]
It looks like that one was already visible. It looks everything on the DM page is already visible. It seems like the players can connect and the is able to load the maptools assets but that loading framework overlay will not go away and it stops them from doing anything.

**Comment by Sigmundx** [comment]
A little more information, my players are able to interact with everything, but there is just that overlay in the way like the second picture above in the original post.

**Comment by Sigmundx** [comment]
Okay with a little testing with my brother we figured out that the overlay errors out unless you have every Lib: visible to players. Now they can load in without that loading framework overlay in the way.

Hope this helps!

**Comment by Pandemicland559** [comment]
Yeah, I fixed it by showing a notebook i had in the wrong place. Thanks for all the help
